core/modules/ckeditor5/js/ckeditor5.js

```diff
@@ -34,7 +34,7 @@
       // The source element value is managed manually to apply code formatting.
       editorSettings.updateSourceElementOnDestroy = false;
 
-      editorSettings.licenseKey = '';
+      editorSettings.licenseKey = 'GPL';
```

Comment:
Eventually we might rather expose this setting via PHP alter hook by setting it in function ckeditor5_get_config() instead of hardcoding it in js.
But I'm not sure if anyone needs that, anyway. :wink: